sam-app/hello_world/app.py

```diff
@@ -1,9 +1,152 @@
 import json
```

Comment:
sam init created a default folder called sam-app. Everything inside sam-app should be in your root repo folder. Also, hello_world is just an example name. You should change it to something meaningful like lambdametrics.